### DISCUSSION TITLE
Question: User with shared credentials + License management.

### DISCUSSION THREAD

**Discussion by vinay-sd17** [Glench/ExtPay#91] [Q&A]
Quick question:

What happens if a user purchases with his email & at the same time shares his email credentials with other people? 

1. Will ExtPay still consider it as a valid license (1 user with multiple devices)? 
2. If so (1), And If I want to block 1 installation + license per device, is it possible?

**Comment by surtic86** [comment]
Right now you can only activate a Instance with your E-Mail 10 Times. After that the Users can't activate any new Instances.

The User will get the Message: This email has been used to many Times.

**Reply by vinay-sd17** [reply]
Oh, that's nice to hear. 

Can we have it as a configuration in the ExtPay dashboard instead of 10 by default?
